src/client/components/user-context.tsx

```diff
@@ -20,29 +24,48 @@ export const UserContextProvider = ({children}) => {
   };
 
   useEffect(() => {
-      // refresh cookie
-      fetchJson(`/refresh`, (err, res) => {
```

Comment:
Did this not work for your needs? What was missing?

Reply:
The problem here was with res.redirect() from the server side, it ended up with an error parsing html as json.